adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -351,7 +354,7 @@ def start_scan_networks(self):
 
     def get_scan_networks(self):
         """The results of the latest SSID scan. Returns a list of dictionaries with
-        'ssid', 'rssi' and 'encryption' entries, one for each AP found"""
+        'ssid', 'rssi', 'encryption', bssid, and channel entries, one for each AP found"""
```

Comment:
Super tiny nitpick, add single tick's (`) around `bssid` and `channel` for the docstring.